File: generated-sources/php-ze-ph/mojang-api/src/App/DTO/InlineObject1.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Request Mojang to download a skin from an URL and apply to the player
 */
class InlineObject1
{
    /**
     * @DTA\Data(field="model", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\SkinModel::class})
     * @DTA\Validator(name="Dictionary", options={"type":\App\DTO\SkinModel::class})
     * @var \App\DTO\SkinModel
     */
    public $model;
    /**
     * The URL which Mojang servers will download and apply the skin
     * @DTA\Data(field="url")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $url;
}

File: generated-sources/dart-jaguar/mojang-api/lib/model/inline_object1.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:openapi/model/skin_model.dart';
part 'inline_object1.jser.dart';

class InlineObject1 {
  
  @Alias('model')
  final SkinModel model;
   /* The URL which Mojang servers will download and apply the skin */
  @Alias('url')
  final String url;
  

  InlineObject1(
    

{
     this.model = null,  
    
     this.url = null 
    }
  );

  @override
  String toString() {
    return 'InlineObject1[model=$model, url=$url, ]';
  }
}

@GenSerializer()
class InlineObject1Serializer extends Serializer<InlineObject1> with _$InlineObject1Serializer {

}
